examples/templates/docker/main.tf

```diff
@@ -187,6 +187,9 @@ resource "docker_container" "workspace" {
     volume_name    = docker_volume.home_volume.name
     read_only      = false
   }
+  capabilities {
+    add = ["CAP_NET_ADMIN"]
+  }
```

Comment:
IDK if we should default to having this enabled in the example Docker template because of the potential security implications. I know that this only lets it impact it's own network adapter but potentially the container might be able to screw with routing on the whole host in some scenarios.

If we do want to keep it we should add a comment explaining that it reduces security.

Reply:
Maybe I can just leave it in as commented, with instructions to uncomment if desired? I mostly just wanted one of our examples to show how to do it.